tests/tst_calendarevent/tst_calendarevent.cpp

```diff
@@ -78,6 +83,9 @@ void tst_CalendarEvent::initTestCase()
     plugin->initializeEngine(engine, "foobar");
     calendarApi = new CalendarApi(this);
 
+    // Use test plugins, instead of mKCal ones.
+    qputenv("MKCAL_PLUGIN_DIR", "plugins");
```

Comment:
Well, that's the purpose of the environment variable. Except if I miss your point, the purpose is not to look into the full system path. Like that I can move the `tst_calendarevent` and the `plugins/libtestplugin.so` around and run it in my home directory.

Actually, it would have been much simpler to put something `qputenv("MKCAL_PLUGIN_DIR", ".");`, but the service loader in mKCal is trying to load all the file in the directory, and my test directory has a lot of files... So I decided for a subdir and install the plugin accordingly, see the .pro file. Do you think it could be an issue on the test platform ?

Reply:
I meant having here /opt/tests/nemo-qml-plugins-qt5/calendar/plugins. Would duplicate the path and needs adjustment if someone ever wanted to support some other install directory, but then again I'm not sure does the current testrunner xml work otherwise. The <step> doesn't have "cd" so I'm not sure what's the working directory when the tests gets executed.

Reply:
Ok, then, I propose to setup this value in the `tests.xml` file and adjust a bit the test code to export the relative path only if no environment variable has been set yet.

I've adjusted the code accordingly. What do you think ?